python/sglang/api.py

```diff
@@ -35,12 +36,14 @@ def decorator(func):
 def Runtime(*args, **kwargs):
     # Avoid importing unnecessary dependency
     os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
-    from sglang.srt.server import Runtime
 
-    return Runtime(*args, **kwargs)
+    return SRTRuntime(*args, **kwargs)
 
 
-def set_default_backend(backend: BaseBackend):
+def set_default_backend(backend: Union[BaseBackend, SRTRuntime]):
+    if isinstance(backend, SRTRuntime):
+        backend = backend.endpoint
```

Comment:
This change only works for `set_default_backend`.

We have some other use cases where we do not call `set_default_backend`, but directly use things like
`program.run(backend=backend)`